packages/editor/src/components/post-title/index.native.js

```diff
@@ -55,8 +69,9 @@ class PostTitle extends Component {
 			<TextInput
 				blurOnSubmit={ true }
 				textAlignVertical="top"
-				multiline
-				numberOfLines={ 0 }
+				multiline={ false }
+				onSubmitEditing={ this.onEnter }
```

Comment:
For the comment above to work, this would become `onSubmitEditing={ this.props.onEnterPress }`

Reply:
Nice find @diegoreymendez!

Tested here as well, and worked without problems 👍 I've updated this PR and main PR gb-mobile side.